src/views/modal_handler.rs

```diff
@@ -113,50 +143,90 @@ impl DisplayModal {
     }
 
     pub fn view(&self) -> Element<Message> {
-        let mut button_row = Row::new();
-        let mut text_style = TextStyle {
-            text_color: Color::new(0.447, 0.624, 0.812, 1.0),
-        };
-
-        if self.is_warning {
-            text_style = TextStyle {
-                text_color: Color::new(0.988, 0.686, 0.243, 1.0),
-            };
-            button_row = button_row.push(
-                button("Exit without saving")
-                    .on_press(Message::ModalHandle(ModalMessage::ExitApp))
-                    .style(MODAL_CANCEL_BUTTON_STYLE.get_button_style()),
-            ); // Exits the application
-            button_row = button_row
-                .push(
-                    button("Return to app")
+        match &self.modal_type {
+            Some(ModalType::Warning {
+                title,
+                body,
+                load_config,
+            }) => {
+                let mut button_row = Row::new();
+                let text_style = TextStyle {
+                    text_color: Color::new(0.988, 0.686, 0.243, 1.0),
+                };
+
+                if *load_config {
```

Comment:
There is a lot of similar code in both branches of if-then-else

Can't just set variables for "exit button text" and "stay button text" and then have all the view code be just one?